index.md

```diff
@@ -56,20 +56,18 @@ __Technical information__
 
 ## Project and community
 
-Charmed HPC is a project of the [Ubuntu High-Performance Computing
+Charmed HPC is an open-source project of the [Ubuntu High-Performance Computing
```

Comment:
IIRC Canonical's standard practice is to not hyphenate "open source". I remember being lectured about this after I wrote a blog post for ubuntu.com :sweat_smile: 

```suggestion
Charmed HPC is an open source project of the [Ubuntu High-Performance Computing
```